apache/datadog_checks/apache/apache.py

```diff
@@ -37,7 +37,7 @@ class Apache(AgentCheck):
         'connect_timeout': {'name': 'connect_timeout', 'default': 5},
     }
 
-    VERSION_REGEX = re.compile(r'^Apache/([0-9]+\.[0-9]+\.[0-9]+)( \(.*\))?$')
+    VERSION_REGEX = re.compile(r'^Apache/(\d+(?:\.\d+)*)')
```

Comment:
I think it's better to not be too strict, it will make the matching more resilient/futureproof:

```suggestion
    VERSION_REGEX = re.compile(r'Apache/(\d+(?:\.\d+)*)')
```